themes/navy/languages/fr.yml

```diff
@@ -0,0 +1,69 @@
+menu:
+  docs: Docs
+  api: API
+  news: Nouvelles
+  plugins: Plugins
+  themes: Thèmes
+  search: Chercher
+
+index:
+  get_started: Commercer
```

Comment:
I guess there is a typo here. You wanted to write `Commencer`?

Reply:
guest what ;)
commencer instead of commercer ?
you might ask for the ç as commençer
but commencer is as french as i am
https://www.larousse.fr/dictionnaires/francais/commencer/17470
![image](https://user-images.githubusercontent.com/5204724/42978676-8fc1f7c2-8bce-11e8-9333-8645984130a6.png)